libr/core/cmd.c

```diff
@@ -124,14 +124,15 @@ static const char *help_msg_star[] = {
 };
 
 static const char *help_msg_dot[] = {
-	"Usage:", ".[r2cmd] | [file] | [!command] | [(macro)]", " # define macro or load r2, cparse or rlang file",
+	"Usage:", ".[r2cmd] | [file] | [!command] | [(macro)]", "# define macro or interpret r2, cparse, r_lang,"
```

Comment:
this line is too long and splitting it that way makes msvc unhappy

Reply:
0838d715fa7026141a7e126c0e6a7af34675d599 ok?